packages/ember-htmlbars/lib/main.js

```diff
@@ -96,23 +83,4 @@ if (Ember.FEATURES.isEnabled('ember-htmlbars')) {
 
 }
 
-var domHelper = environment.hasDOM ? new DOMHelper() : null;
-
-export var defaultEnv = {
-  dom: domHelper,
-
-  hooks: {
-    get: get,
-    set: set,
-    inline: inline,
-    content: content,
-    block: block,
-    element: element,
-    subexpr: subexpr,
-    component: component,
-    attribute: attribute,
-    concat: concat
-  },
-
-  helpers: helpers
-};
+export var defaultEnv;
```

Comment:
I do not believe this is used (in `render-view` you import directly from `ember-htmlbars/env` not `ember-htmlbars`).  That also means you don't need the import above either...